docs/logo/fizzy_logo.svg

```diff
@@ -2,22 +2,27 @@
 <!-- Generator: Adobe Illustrator 24.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 600 600" style="enable-background:new 0 0 600 600;" xml:space="preserve">
+<title>Fizzy logo (blue)</title>
 <style type="text/css">
 	.st0{fill:#654FF0;}
 	.st1{fill:#EDC433;}
 	.st2{fill:#FFFFFF;}
 </style>
-<g id="Layer_2_1_">
-	<rect x="-12" y="-11" class="st0" width="624" height="618"/>
+<!-- Background -->
+<g id="Layer_2">
+	<rect x="0" y="0" class="st0" width="600" height="600"/>
 </g>
+<!-- Cup -->
 <g id="Layer_3">
-	<polygon class="st1" points="391,552 208,552 149,250 451,250 	"/>
+	<polygon class="st1" points="391,552 208,552 149,250 451,250"/>
```

Comment:
These numbers would be more perfectly centered
```suggestion
	<polygon class="st1" points="391,550 209,550 149,250 451,250"/>
```
(I didn't try to move letter F, but with this change visually I don't see any imperfection)

Reply:
Added. Is it good after this change?